docs/src/user-guide/formatters/index.md

```diff
@@ -57,16 +57,30 @@ function addOne(i) {
 }
 ```
 
-## Output Examples
+Tests the formatters with the CLI:
+
+```shell
+npx eslint --formatter <Add formatter here> fullOfProblems.js
+```
+
+## Built-In Formatter Options
 
 ### checkstyle
 
+TODO: quick overview
```

Comment:
Another one to pass the data to a system that expects a specific format,

Context: https://github.com/eslint/eslint/issues/14 (not much context there though, here's [checkstyle docs](https://checkstyle.sourceforge.io/)).

Reply:
Back in the day, a lot of Jenkins plugins expected results in Checkstyle format. It's not nearly as popular as it was but there are still a few Jenkins plugins that expect that format:

https://plugins.jenkins.io/ui/search?query=checkstyle